homeassistant/components/withings/common.py

```diff
@@ -1030,6 +1030,23 @@ async def async_get_data_manager(
     if const.DATA_MANAGER not in config_entry_data:
         profile = config_entry.data.get(const.PROFILE)
 
+        if CONF_CLOUDHOOK_URL not in config_entry_data:
```

Comment:
What's a `.` vs a `_` between variables, huh? 😅

Thanks for catching that, should be fixed now, but it shows that I can't test this properly :/

Reply:
Not being able to test it, makes this kinda hard to accept as a PR, to be honest.
Maybe add unit tests to verify it's working instead?

Reply:
The one instance where I can test withings is a friend's that has a publicly routable address, but not a Cloud subscription, that's why I couldn't test that behaviour easily.

I'll see what I can do for the tests — there's not much to look at that tests those cloudhook URLs but I guess I can borrow from the plaato config flow tests.

I mostly wanted to have something for those affected by #72614 since to fix one setup I broke the other 😣

Reply:
How can I help test this?  I am affected by #72614 but I do not have a cloud subscription.  I access my Home Assistant via my own domain name and SSL certificate.

Reply:
I have a cloud subscription if I can help test! Looks like I probably opened a duplicate issue here: https://github.com/home-assistant/core/issues/73200

Reply:
Sorry folks, I've been a little [otherwise occupied](https://flameeyes.blog/2022/06/14/got-covid-it-sucks/) so I didn't pay enough attention here.

If you're comfortable with applying this change to your installation, you should be able to apply the diff cleanly on top of any 2022.6.x release, then restarting HA.

Reply:
Ahh I got it sorted! The patch didn't have the changes mentioned here: 
https://github.com/home-assistant/core/pull/73228#discussion_r892421103

Manually fixing that resolves the KeyError. I am still getting the first MockRequest here though - it seems like withings is still making requests to the old webhook id (or maybe the incorrect endpoint, but still the correct webhook id?). I'm really not sure of the inner workings of the webhooks/home assistant configs yet.